binary_transparency/firmware/README.md

```diff
@@ -0,0 +1,88 @@
+Firmware Transparency
+=====================
+
+This directory contains a description (TODO: implement) of how to apply transparency patterns and principles to the problem of firmware updates.
+
+Background
+----------
+
+**Firmware is ubiquitous**; it's in your phone, watch, TV, alarm clock, baby
+monitor, WiFi devices, possibly even your lightbulbs if you have LED lamps. In
+any given desktop PC there's the BIOS/UEFI type firmware that many people are
+familiar with, but there are also scores of other hidden firmware _blobs_
+running on small controllers which power things like management engines,
+keyboards, network cards, hard-disks/SSDs. The list goes on.
+
+**Firmware is powerful**, it runs at the highest priviledge level possible and
+is often the bedrock on which the security story of the devices it powers
+depends.
+
+It is also often almost entirely invisible and inscrutable, and in many cases
+has been
+[shown to be insecure and vulnerable](https://eclypsium.com/2020/2/18/unsigned-peripheral-firmware).
+
+Today, the best-in-class vendors who supply the firmware also provide an update
+framework which verifies the integrity and authenticity of firmware updates
+before allowing them to be installed.
+
```

Comment:
The intent of this bit was narrative in nature - "oh noes! question, question, QUESTION?!" with the next section header being the answer :)
I'll leave it alone for now since I expect all of this to get seriously hacked on in following PRs.